velox/connectors/hive/storage_adapters/s3fs/tests/S3FileSystemTest.cpp

```diff
@@ -101,6 +101,16 @@ class S3FileSystemTest : public testing::Test {
 
 std::shared_ptr<MinioServer> S3FileSystemTest::minioServer_ = nullptr;
 
+std::string removeRequestID(std::string input) {
```

Comment:
Why do we need to remove the RequestID for the test.

Reply:
because the RequestID is an random generated string, we would not get it before, but after rebasing against master, we would remove because VELOX_ASSERT_THROW uses find method to match errorMessage.